### ISSUE TITLE
The screen is blank after startup.

### ISSUE THREAD

**Issue by kyonefq** [sytone/obsidian-remote#105]
![image](https://github.com/user-attachments/assets/bdd51545-dbd3-4586-88dd-db84189bd160)

**Comment by jermany182** [comment]
Same issue here, did you ever figure it out?

**Comment by maneeshsethi** [comment]
Same for me, any ideas?

**Comment by jermany182** [comment]
Nah man, I shut it down and moved on haha. I had it working once a couple years ago but I don't remember what I did differently.

**Comment by jermany182** [comment]
@maneeshsethi Overkill solution maybe, but I was planning on setting Kasm back up, and LinuxServer.io has a repository on there that has Obsidian.  Updated regularly too.

**Comment by luxuwin** [comment]
same here

**Comment by EagerCookie** [comment]
I had the same problem - "white rectangle"
Tried to forward video device via command --device /dev/dri/renderD128:/dev/dri/renderD128
And UI appeared!
... but after a few moments, all disappeared and now nothing is working at all...

**Comment by shenweiyan** [comment]
same here

**Comment by Gireza** [comment]
Yes, exactly the same, the last log on the docker container is : 
obsidian-remote  | guacd[430]: ERROR:   User is not responding.
obsidian-remote  | guacd[430]: INFO:    User "@5abb7823-4942-4f18-a4d1-653f9fa0d781" disconnected (0 users remain)
obsidian-remote  | guacd[430]: INFO:    Last user of connection "$9c914490-b87f-49ce-b45e-f8155f7d1086" disconnected
obsidian-remote  | guacd[200]: INFO:    Connection "$9c914490-b87f-49ce-b45e-f8155f7d1086" removed.

**Comment by YuZhang-GitHub** [comment]
same here

**Comment by CobalticBlue** [comment]
Same here, any ideas?

**Comment by quyendaoqt707** [comment]
For those who have met this issue:

1. Don't use the `ghcr.io/sytone/obsidian-remote:latest` image, this is too out of date. Use `sytone/obsidian-remote:latest` (DockerHub) instead (new Obsidian and Kasm version)
2. Make sure the container can access and write on /config volume (which should be empty at first startup). 
3. Ensure you already wiped the config volume when switching the image to `sytone/obsidian-remote:latest`. The previous config generated by the ghcr image may cause `Failed to execute child process "/squashfs-root/obsidian" (No suchfile or directory)` when trying to start Obisidan via the right-click menu.
